tests/fixtures/basic_report_no_default_features.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites name="my-test-run" tests="7" failures="2" errors="1" time="42.235">
+    <testsuite name="testsuite0" tests="7" disabled="1" errors="1" failures="2">
+        <properties>
+            <property name="env" value="FOOBAR"/>
+        </properties>
+        <testcase name="testcase0">
+            <system-out>testcase0-output</system-out>
+        </testcase>
+        <testcase name="testcase1" time="4.242">
+            <failure message="testcase1-message">this is the failure description</failure>
+            <system-err>some sort of failure output</system-err>
+        </testcase>
+        <testcase name="testcase2" time="0.000">
+            <error type="error type">testcase2 error description</error>
+        </testcase>
+        <testcase name="testcase3" assertions="20">
+            <skipped message="skipped message" type="skipped type"/>
+            <system-out>testcase3 output</system-out>
+            <system-err>testcase3 error</system-err>
+        </testcase>
+        <testcase name="testcase4" time="661.661">
+            <flakyFailure type="flaky failure type">this is a flaky failure description</flakyFailure>
+            <flakyError type="flaky error type">flaky error description
+                <stackTrace>flaky stack trace</stackTrace>
+                <system-out>flaky system output</system-out>
+                <system-err>flaky system error with [34mANSI escape codes[39m</system-err>
```

Comment:
Without the ansi-escape feature flag, ANSI escape codes are just added to the XML. It looks like there is actually no additional escaping necessary to be valid XML (but if it were, quick-xml would take care of it).